certified-connectors/RAPID Platform/apiProperties.json

```diff
@@ -0,0 +1,65 @@
+{
+	"properties": {
+		"connectionParameters": {
+			"tenantName": {
+				"type": "string",
+				"uiDefinition": {
+					"constraints": {
+						"required": "true"
+					},
+					"description": "Specify name of your tenant",
+					"displayName": "Tenant Name",
+					"tooltip": "Provide your tenant name"
+				}
+			},
+			"siteName": {
+				"type": "string",
+				"uiDefinition": {
+					"constraints": {
+						"required": "true"
+					},
+					"description": "Specify name of your Site",
+					"displayName": "Site Name",
+					"tooltip": "Provide your site name"
+				}
+			},
+			"token": {
+				"oAuthSettings": {
+					"clientId": "b427679a-323d-4beb-b679-fa1efa0eed89",
+					"customParameters": {
+						"loginUri": {
+							"value": "https://login.windows.net"
+						},
+						"resourceUri": {
+							"value": "cd5db0ec-1419-4ae6-9434-21cfb83fc42d"
+						},
+						"tenantId": {
+							"value": "common"
+						}
+					},
+					"identityProvider": "aad",
+					"properties": {
+						"IsFirstParty": "False"
+					},
+					"redirectMode": "Global",
+					"redirectUrl": "https://global.consent.azure-apim.net/redirect",
+					"scopes": []
+				},
+				"type": "oauthSetting"
+			}
+		},
+		"iconBrandColor": "#00b3f1",
```

Comment:
@MattFranklin-Rapid Please make sure that you are following the [instructions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#icon) for uploading icon while you make submission in in the certification. The `iconBrandColor` should be the background color of your connector's icon.

Reply:
I can confirm the iconBrandColour key matches the background of the icon but am unsure of how to submit the icon itself. Currently there is a gitignore for icon.png and icon.jpg with a clear comment to not submit icons. Should the icon be added here via the pull request or through some other medium?

Reply:
@diegobernardes Icon is not need to add in the PR